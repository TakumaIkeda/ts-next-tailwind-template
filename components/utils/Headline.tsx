import React from "react";
import classNames from "classnames";

const Headline: React.FC = ({ children }) => {
  const className = classNames("text-2xl", "font-bold");
  return <h1 className={className}>{children}</h1>;
};

export default Headline;
