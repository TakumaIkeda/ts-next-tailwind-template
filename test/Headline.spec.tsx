import React from "react";
import renderer from "react-test-renderer";
import Headline from "../components/utils/Headline";

test("AppTitle", () => {
  const component = renderer.create(<Headline />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
