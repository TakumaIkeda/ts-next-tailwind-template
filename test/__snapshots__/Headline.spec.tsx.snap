// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTitle 1`] = `
<h1
  className="text-2xl font-bold"
/>
`;
